cmd/beacon-chain/flags/BUILD.bazel

```diff
@@ -12,6 +12,7 @@ go_library(
     visibility = [
         "//beacon-chain:__subpackages__",
         "//cmd/beacon-chain:__subpackages__",
+        "//shared/gateway:__pkg__",
```

Comment:
Needed to expose this for testing purposes, the flags for the context needed for a gateway.